src/Parser/Parser.hs

```diff
@@ -537,7 +538,7 @@ parseFromFile _p fname = do
 -- Still takes a file name so as to provide a reasonable debug message if parsing fails
 -- This will likely be something general, such as Prelude or Gamefile
 parseFromText :: Parser a -> String -> String -> Either ParseError a
-parseFromText _p fn content = parseAll _p fn content
+parseFromText _p fn contents = parseAll _p fn contents
```

Comment:
Man, sometimes this gets me...the subtle addition of functions into the local scope because of record syntax, honestly maddening at times -_-. I wish there was a way to keep the name but prefix them, w/out having to move them into an entirely separate file to try and isolate them.